### ISSUE TITLE
Chat Crashing - "String contained an illegal UTF-16 sequence"

### ISSUE THREAD

**Issue by ericzhou27** [GetStream/stream-chat-react-native#979]
Hello, one of our chat rooms is crashing on launch with this error when running it locally. We have no idea why, but it looks like an issue under the hood with some message possibly? Please advise. All other chat rooms are working.

![simulator_screenshot_249F3E6B-9136-4DD2-9A4B-7F414D3208FA](https://user-images.githubusercontent.com/43918217/140802638-20ea8ed9-3ea2-4b41-b460-03978afb61da.png)

**Comment by ericzhou27** [comment]
I was able to fix it from the moderation dashboard by deleting a few messages. One of the messages I deleted had an image - I'm assuming that was the problem? We need to make sure that this doesn't happen again. Is there something we have to add to the blocklist or something?

**Comment by madsroskar** [comment]
Hi @ericzhou27.

This is likely to be a link in a message containing a character that isn't allowed, but it's a bit tricky to know which it would be - did you happen to notice whether any of the messages contained link(s) at all?

We will look into this, and get back to you once we know more.

Thanks for raising the issue!

**Comment by ericzhou27** [comment]
Yep, I've also been speaking with support - they have my channel ID, app ID, and other relevant information. I had the same suspicion as you - I deleted the most recent message and a message containing an image, and it started working again. The image was of a pigeon. I don't think there were any links in the message but I assume the link was to the relevant picture.

**Comment by madsroskar** [comment]
@ericzhou27  I'm having some trouble reproducing this issue, have you been able to see another message that makes this happen since?

**Comment by ericzhou27** [comment]
Hi @madsroskar, I'm in communication with support about querying the original message, which I only soft-deleted. Doing so should help get to the bottom of this!

**Comment by vanGalilea** [comment]
Closing it due not being able to reproduce it.
